packages/shared_preferences/shared_preferences_tool/lib/src/ui/keys_panel.dart

```diff
@@ -107,15 +99,44 @@ class _KeysPanelState extends State<KeysPanel> {
   }
 }
 
+class _ToolbarAction extends StatelessWidget {
```

Comment:
Looks good. can you add a todo on this class to use ToolbarAction from devtools_app_shared if it is exposed, and link to this issue? https://github.com/flutter/devtools/issues/7793